ManagedInjector.CLI/ManagedInjector.CLI.csproj

```diff
@@ -12,7 +12,11 @@
     </ItemGroup>
 
     <ItemGroup>
+      <PackageReference Include="AsmResolver.DotNet" Version="4.5.1" />
```

Comment:
These new package references aren't needed, since you're not using them anywhere.